tests/test_emnlp_conference.py

```diff
@@ -341,90 +341,70 @@ def test_submit_papers(self, test_client, client, openreview_client, helpers):
             'EMNLP/2023/Conference/Submission5/Authors'
         ]
 
-        ## withdraw submission
-        withdraw_note = test_client.post_note_edit(invitation='EMNLP/2023/Conference/Submission5/-/Withdrawal',
+        ## delete and revert submission
+        revision_note = test_client.post_note_edit(invitation='EMNLP/2023/Conference/Submission5/-/Revision',
             signatures=['EMNLP/2023/Conference/Submission5/Authors'],
             note=openreview.api.Note(
-                content={
-                    'withdrawal_confirmation': { 'value': 'I have read and agree with the venue\'s withdrawal policy on behalf of myself and my co-authors.' },
+                ddate = openreview.tools.datetime_millis(datetime.datetime.utcnow()),
+                content = {
+                    'title': { 'value': 'Test Paper Title 5' },
+                    'authors': submissions[4].content['authors'],
+                    'authorids': submissions[4].content['authorids'],
+                    'abstract': { 'value': 'This is a test abstract 5' },
+                    'keywords': { 'value': ['machine learning', 'nlp'] },
+                    'submission_type': { 'value': 'Regular Long Paper' },
+                    'pdf': { 'value': '/pdf/' + 'p' * 40 +'.pdf' } #fields not in the submission need to be added to delete submission?
```

Comment:
I found this issue yesterday. EMNLP added fields to the revision form that were required (that were not in the submission). In order to delete a submission, the authors need to fill out all these new, required fields, but it seems weird to ask them to if they are deleting their submission.

Reply:
@celestemartinez I think we would need to create an invitation that let authors delete their submissions between the abstract and full paper deadline. 

The invitation can specify the ddate field only, what do you think?

Reply:
Sounds good, I can try this! This means I would remove the ddate field from the submission invitation or would we keep that as well?

Reply:
let's remove the ddate from the revision invitation so we don't have the issue that EMNLP has. So if the invitation has the ddate field only, it should render the trash icon and it shouldn't appear in the edit dropdown, can you verify that?